package.json

```diff
@@ -3,7 +3,7 @@
   "name": "@purple-dot/browser",
   "description": "Purple Dot Browser SDK",
   "license": "Apache-2.0",
-  "version": "0.2.3",
+  "version": "0.2.4",
```

Comment:
Is this warranted?

Reply:
dropping node versions is normally considered a breaking change, so you'd probably jump to 1.0.0

Reply:
On Slack we've established that we don't actually need this.